tests/runtime/out_elasticsearch.c

```diff
@@ -347,6 +347,47 @@ void flb_test_write_operation_upsert()
     flb_destroy(ctx);
 }
 
+void flb_test_null_index()
```

Comment:
Does this test need to be added into [TEST_LIST](https://github.com/tkennedy1-godaddy/fluent-bit/blob/8107bf412e87c41cf3d38eac445845c315a3653b/tests/runtime/out_elasticsearch.c#L1002)?